README.md

```diff
@@ -13,6 +13,7 @@ Kill Bill reusable Java components:
 * **metrics**: annotation-based metrics
 * **queue**: provides persistent bus events and notifications
 * **skeleton**: framework to build web services
+* **utils**: provides utility methods for core Java(tm) classes (lang, IO, collection)
```

Comment:
Can you update the `Kill Bill compatibility` section below (so that we have the `artifactId` handy), as well as add the new dependency in https://github.com/killbill/killbill-oss-parent?

Reply:
Sorry for the confusion, I meant the Usage section: https://github.com/killbill/killbill-commons#usage

Reply:
fixed usage section, but

> as well as add the new dependency in https://github.com/killbill/killbill-oss-parent?

I think this is should be done separately in `killbill-oss-parent` ? I'll send PR.